Packs/QRadar/Integrations/QRadar_v3/QRadar_v3_test.py

```diff
@@ -1645,3 +1649,99 @@ def test_qradar_reference_set_value_upsert_command_continue_polling_with_connect
     result = qradar_reference_set_value_upsert_command(args, client=client, params=api_version)
     # make sure when status is COMPLETED that outputs are returned
     assert result.outputs
+
+@pytest.mark.parametrize('command_func, endpoint, id', [
```

Comment:
as discussed, lets refactor the section of code we are interested in testing here into its own function and test that function instead